plugins/dell-dock/fu-plugin-dell-dock.c

```diff
@@ -81,10 +81,12 @@ fu_plugin_dell_dock_probe(FuPlugin *plugin, FuDevice *proxy, GError **error)
 		instance_id_mst = DELL_DOCK_VMM6210_INSTANCE_ID;
 	else
 		instance_id_mst = DELL_DOCK_VM5331_INSTANCE_ID;
+	fu_device_add_instance_id(FU_DEVICE(mst_device), instance_id_mst);
 	instance_guid_mst = fwupd_guid_hash_string(instance_id_mst);
 	fu_device_add_guid(FU_DEVICE(mst_device), instance_guid_mst);
```

Comment:
Do you still need these two lines? If so, LGTM.

Reply:
Yes, they are needed.